docs/legal/cloud_subprocessors.md

```diff
@@ -19,4 +19,5 @@ Service:
 | Amazon Web Services                | Cloud Hosted Infrastructure and Data Hosting         | United States      |
 | Google LLC                         | Email Hosting, document storage & Video Conferencing | United States      |
 | Slack Technologies, LLC            | Instant messaging and document transfer              | United States      |
-| Functional Software, Inc. (Sentry) | Error capturing & reporting                          | United States      |
+| Buttondown, LLC                    | Email broadcasting platform                          | United States      |
+| Stripe, Inc.                       | Payment processing platform                          | United States      |
```

Comment:
Note that the stripe entity changes depending on which region of the world you're in
https://stripe.com/en-de/legal/service-providers

So there are EU and UK specific entities, for example. @samuelcolvin  any thoughts on what we want to do here?

Reply:
As discussed, US based is the correct choice here for now.